zavod/zavod/sentry.py

```diff
@@ -188,9 +192,8 @@ def _get_event_and_hint(
             event["tags"] = {
                 key: event_dict[key] for key in self.tag_keys if key in event_dict
             }
-        # Because we push the dataset into the Event.logger field (see above), we push the logger field into the tags.
         if "logger" in event_dict:
-            event["tags"]["python_logger"] = event_dict["logger"]
+            event["logger"] = event_dict["logger"]
```

Comment:
watch out we're still setting this on line 176 too

Reply:
Urgh, sorry. Good catch!